config/telink/app/zephyr.conf

```diff
@@ -122,6 +118,9 @@ CONFIG_SETTINGS_NVS_SECTOR_SIZE_MULT=8
 # Enable NVS lookup cache
 CONFIG_NVS_LOOKUP_CACHE=y
 
+# Enable extended discovery
+CONFIG_CHIP_EXTENDED_DISCOVERY=y
```

Comment:
Should this be on by default for the platform?  Generally speaking, extended discovery is only useful for some kinds of devices...

Reply:
Prefer to use by default for Telink platform as well as other vendors did.
If can be disabled by developer for any specific app if needed.

Reply:
Revert this change on current PR.